### PR TITLE
ticdc: fix pulsar sink documentation error

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16587
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-pulsar.md [hunk 1]
```diff
@@ -136,6 +140,32 @@ send-timeout=30
...
+```
+
+Configuration：
+
+```toml
+[sink.pulsar-config]
+tls-trust-certs-file-path="/data/pulsar/tls-trust-certs-file"
+```
+
+If the `tlsRequireTrustedClientCertOnConnect=true` parameter is configured for your Pulsar server, you need to configure the `tls-key-file-path` and `tls-certificate-file` parameters in the changefeed configuration file. For example:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:160]:
```suggestion
If the `tlsRequireTrustedClientCertOnConnect=true` parameter is configured for your Pulsar server, you also need to configure the `tls-key-file-path` and `tls-certificate-file` parameters in the changefeed configuration file. For example:
```

**Review by asddongmen — APPROVED** [review]

File: ticdc/ticdc-sink-to-pulsar.md [hunk 2]
```diff
@@ -99,12 +99,16 @@ token-from-file="/data/pulsar/token-file.txt"
...
 auth-tls-certificate-path="/data/pulsar/certificate"
-# The private key path for Pulsar TLS encrypted authentication.
+# The private key path on the client, which is required when Pulsar enables the mTLS authentication.
 auth-tls-private-key-path="/data/pulsar/certificate.key"
-# Path to trusted certificate file of the Pulsar TLS encrypted authentication.
+# The path to the trusted certificate file of the Pulsar TLS authentication, which is required when Pulsar enables the mTLS authentication or TLS encrypted transmission.
 tls-trust-certs-file-path="/data/pulsar/tls-trust-certs-file"
+# The path to the encryption private key on the client, which is required when Pulsar enables TLS encrypted transmission.
+tls-key-file-path="/data/pulsar/tls-key-file"
+# The path to the encryption certificate file on the client, which is required when Pulsar enables TLS encrypted transmission.
```
Comment (lilin90) [inline ticdc/ticdc-sink-to-pulsar.md:110]: How about "encrypted"?
```suggestion
# The path to the encrypted private key on the client, which is required when Pulsar enables TLS encrypted transmission.
tls-key-file-path="/data/pulsar/tls-key-file"
# The path to the encrypted certificate file on the client, which is required when Pulsar enables TLS encrypted transmission.
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 3]
```diff
@@ -136,6 +140,32 @@ send-timeout=30
 * You need to specify the `protocol` parameter when creating a changefeed. Currently, only the `canal-json` protocol is supported for replicating data to Pulsar.
 * The `pulsar-producer-cache-size` parameter indicates the number of producers cached in the Pulsar client. Because each producer in Pulsar can only correspond to one topic, TiCDC adopts the LRU method to cache producers, and the default limit is 10240. If the number of topics you need to replicate is larger than the default value, you need to increase the number.
 
+### TLS encrypted transmission
+
+Starting from v7.5.1 and v8.0.0, TiCDC supports TLS encrypted transmission for Pulsar. The configuration example is as follows:
+
+Sink URI：
```
Comment (lilin90) [inline ticdc/ticdc-sink-to-pulsar.md:147]:
```suggestion
Sink URI:
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 4]
```diff
@@ -136,6 +140,32 @@ send-timeout=30
...
+
+```shell
+--sink-uri="pulsar+ssl://127.0.0.1:6651/persistent://public/default/yktest?protocol=canal-json"
+```
+
+Configuration：
```
Comment (lilin90) [inline ticdc/ticdc-sink-to-pulsar.md:153]:
```suggestion
Configuration:
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 5]
```diff
@@ -171,32 +201,34 @@ The following is a sample configuration when you use token authentication with P
...
     ```toml
     [sink.pulsar-config]
-    # Certificate path of the Pulsar TLS encrypted authentication
+    # Certificate path of the Pulsar mTLS authentication
     auth-tls-certificate-path="/data/pulsar/certificate"
-    # Private key path of the Pulsar TLS encrypted authentication
+    # Private key path of the Pulsar mTLS authentication
     auth-tls-private-key-path="/data/pulsar/certificate.key"
-    # Path to trusted certificate file of the Pulsar TLS encrypted authentication
+    # Path to trusted certificate file of the Pulsar mTLS authentication
```
Comment (lilin90) [inline ticdc/ticdc-sink-to-pulsar.md:220]:
```suggestion
    # Path to the trusted certificate file of the Pulsar mTLS encrypted authentication
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**